部分单元测试在本地无预配置好H2环境的新环境编译测试

**描述您遇到的bug**  
部分单元测试没有正确配置H2数据库（继承现有测试类AbstractControllerTest或者DatabaseBasedTest）导致测试不通过。

```
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   OperationLogServiceTest.queryByCategoryTest ? IllegalState Failed to load Appl...
[ERROR]   OperationLogServiceTest.queryByCreatorTest ? IllegalState Failed to load Appli...
[ERROR]   OperationLogServiceTest.queryByNoExistingCreatorTest ? IllegalState Failed to ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase1 ? IllegalState Failed to load ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase2 ? IllegalState Failed to load ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase3 ? IllegalState Failed to load ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase4 ? IllegalState Failed to load ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase5 ? IllegalState Failed to load ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase6 ? IllegalState Failed to load ...
[ERROR]   PluginInstanceServiceTest.isIpValidityTestCase7 ? IllegalState Failed to load ...
[ERROR]   ProcessDefinitionServiceTest.evaluateRequiredInputParameters ? IllegalState Fa...
[INFO]
[ERROR] Tests run: 29, Failures: 0, Errors: 11, Skipped: 1
```

**如何重现** 
在无预配置本地H2数据库的环境编译安装wecube-core

```bash
cd wecube-platform
mvn test
```

**预期行为** 
部分单元测试不通过。

**截图**  
![image](https://user-images.githubusercontent.com/52971115/66621433-c7fa0e00-ec16-11e9-97ea-fb8314789925.png)


**附加**

--- a/wecube-core/src/test/java/com/webank/wecube/core/service/OperationLogServiceTest.java
+++ b/wecube-core/src/test/java/com/webank/wecube/core/service/OperationLogServiceTest.java
@@ -1,5 +1,6 @@
 package com.webank.wecube.core.service;
 
+import com.webank.wecube.core.DatabaseBasedTest;
 import com.webank.wecube.core.domain.OperationLog;
 import com.webank.wecube.core.dto.OperationLogDto;
 import com.webank.wecube.core.dto.QueryRequest;
@@ -18,7 +19,7 @@ import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 @Slf4j
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class OperationLogServiceTest {
+public class OperationLogServiceTest extends DatabaseBasedTest {
 
     @Autowired
     private OperationLogService operationLogService;

--- a/wecube-core/src/test/java/com/webank/wecube/core/service/PluginInstanceServiceTest.java
+++ b/wecube-core/src/test/java/com/webank/wecube/core/service/PluginInstanceServiceTest.java
@@ -1,5 +1,6 @@
 package com.webank.wecube.core.service;
 
+import com.webank.wecube.core.DatabaseBasedTest;
 import lombok.extern.slf4j.Slf4j;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -12,7 +13,7 @@ import static org.assertj.core.api.Assertions.assertThat;
 @RunWith(SpringRunner.class)
 @SpringBootTest
 @Slf4j
-public class PluginInstanceServiceTest {
+public class PluginInstanceServiceTest extends DatabaseBasedTest {
 
     @Autowired
     PluginInstanceService pluginInstanceService;

--- a/wecube-core/src/test/java/com/webank/wecube/core/service/workflow/ProcessDefinitionServiceTest.java
+++ b/wecube-core/src/test/java/com/webank/wecube/core/service/workflow/ProcessDefinitionServiceTest.java
@@ -1,6 +1,8 @@
 package com.webank.wecube.core.service.workflow;
 
 import com.google.common.collect.Lists;
+import com.webank.wecube.core.DatabaseBasedTest;
+import com.webank.wecube.core.controller.AbstractControllerTest;
 import com.webank.wecube.core.domain.plugin.PluginConfigInterface;
 import com.webank.wecube.core.domain.plugin.PluginConfigInterfaceParameter;
 import com.webank.wecube.core.domain.workflow.ServiceTaskVO;
@@ -31,7 +33,7 @@ import static org.mockito.Mockito.when;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class ProcessDefinitionServiceTest {
+public class ProcessDefinitionServiceTest extends DatabaseBasedTest {
 
     @InjectMocks
     ProcessDefinitionService processDefinitionService;

